recipe/meta.yaml

```diff
@@ -1,47 +1,65 @@
 {% set name = "routingpy" %}
-{% set version = "0.3.3" %}
+{% set version = "1.3.0" %}
+# commit for 1.3.0 tag
+{% set commit = "dc504af4a2445e9db836d3af9781ccc76c3856f2" %}
```

Comment:
why we cannot take it from:
https://github.com/gis-ops/routingpy/archive/refs/tags/1.3.0.tar.gz ?

Reply:
Because then the version will be `None` and setuptools_scm raises an error. I tried to patch but the real issue is that setuptools_scm requires a git repository but https://github.com/gis-ops/routingpy/archive/refs/tags/1.3.0.tar.gz hasn't it.

Reply:
and the git_tag could be 1.3.0, instead of commit hash ?

Reply:
You are right. I've fixed it